preferences-src/src/components/pages/ExtensionPreferences.vue

```diff
@@ -10,18 +10,24 @@
         <div class="repo" v-if="extension.url">
           <div v-if="extension.commit_hash"><i class="fa fa-code-fork fa-fw"></i><span class="text-monospace">{{ extension.commit_hash.slice(0, 7) }}</span></div>
           <div v-if="extension.commit_hash"><i class="fa fa-calendar fa-fw"></i>{{ extension.updated_at.slice(0, 10) }}</div>
-          <div>
-            <a class="text-muted" href @click.prevent="openUrl(extension.url)" :title="extension.url">
+          <div v-if="extension.browser_url">
+            <a class="text-muted" href @click.prevent="openUrl(extension.browser_url)" :title="extension.browser_url">
               <i class="fa fa-external-link"></i> Open repository
             </a>
           </div>
+          <div v-if="!extension.browser_url">
+            <i class="fa fa-copy"></i>
+            <a class="text-muted" href @click.prevent
+             v-clipboard:copy="extension.url"
+             :title="'Click to copy: ' + extension.url">Copy url</a>
+          </div>
```

Comment:
If we have no `browser_url` show a different UI to copy the input URL instead. This is pretty hard to test. It requires a git remote that is not GitHub, GitLab or Codeberg.

(so I tested it by temporarily flipping the `extension.browser_url` and `!extension.browser_url` conditions instead)